client/src/annoMatrix/annoMatrix.ts

```diff
@@ -203,23 +218,23 @@ export default class AnnoMatrix {
   /**
    ** General utility methods
    **/
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  base() {
+  base(): AnnoMatrix {
     /*
     return the base of view, or `this` if not a view.
     */
-    // eslint-disable-next-line @typescript-eslint/no-this-alias --- FIXME: disabled temporarily on migrate to TS.
+    // eslint-disable-next-line @typescript-eslint/no-this-alias --- TODO revist: Possibly remove need for aliasing here, see related comments below on L226.
     let annoMatrix = this;
+    // @ts-expect-error ts-migrate --- TODO revisit:
+    // Compiler is complaining that this could be instantiated with an arbritrary type which AnnoMatrix | null cannot be
+    // assigned to. Possibly move method to recursion pattern to resolve this and issue above (L224).
```

Comment:
+1 - good suggestion.  Perhaps add a property accessor called `getViewOf()` that returns `this` if it isn't a view, else returns `viewOf`

Reply:
* Updated [annoMatrix.ts::base()](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L223)
* Added [annoMatrix.ts::_getViewOf()](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L571)